### ISSUE TITLE
Merge canvas and export to png

### ISSUE THREAD

**Issue by bornakke** [jacomyal/sigma.js#727]
Dear Sigma,

We used to do something like this to merge together scene and edges and export it to a png file. However with the new version of sigma the two images are suddenly placed next to each other. I have tried to debug the error, but I'm really not very good with canvas. 

![download 3](https://cloud.githubusercontent.com/assets/745051/13731486/17ee3c14-e96a-11e5-8a8d-247bcb3b6acf.png)

Do you have any idea as to why this is suddenly happening? Hope Paris is finding you well.

Best
Tobias  

>>>>>>>

var canvases = $('#sigma-container').children();

var ctx0 = canvases[3].getContext('2d');
var ctx1 = canvases[4].getContext('2d');

ctx1.drawImage(canvases[3], 0, 0);
var img    = canvases[4].toDataURL();

var html = '<img width="900px" src="'+img+'"/>';
echo html;

**Comment by Yomguithereal** [comment]
Hello @bornakke. I am not sure to fully understand your example. Do you have it available it somewhere?

**Comment by bornakke** [comment]
Click 'screenshot' at this page:
http://twitterting.cadm.dk/

**Comment by bornakke** [comment]
Shifted back to the old version of sigma which fixed the issue. 

Shifting between the two versions I noticed that the canvas deceleration changes with the two versions.

**New sigma.min.js**
<canvas class="sigma-scene" style="position: absolute; width: 1455px; height: 500px;" width="2910px" height="1000px"></canvas>
<canvas class="sigma-edges" width="2910px" height="1000px" style="position: absolute; width: 1455px; height: 500px;"></canvas>

**Old sigma.min.js**
<canvas class="sigma-edges" width="1455" height="500px" style="position: absolute; width: 1455px; height: 500px;"></canvas>
<canvas class="sigma-scene" width="1455" height="500px" style="position: absolute; width: 1455px; height: 500px;"></canvas>

Any suggestions as to what is going on?

Best
Tobias

**Comment by Yomguithereal** [comment]
There is something fishy indeed. Might come from the recent support of retina displays. Do you test on a mac?

**Comment by bornakke** [comment]
I do. Mac retina.

**Comment by Yomguithereal** [comment]
Can you perchance try on a non-retina display and tell me whether the result is different please?

**Comment by bornakke** [comment]
Yes! It works. So the problem doesn't seem to be that problematic :+1: 

Any suggestions how we should fix it for mac users?

**Comment by Yomguithereal** [comment]
It should just be a matter of tinkering the plugin a bit to find what's amiss.

**Comment by Yomguithereal** [comment]
That's strange. I can't reproduce this bug on master.
